file-uploader/inject/injector.go

```diff
@@ -0,0 +1,52 @@
+//+build wireinject
+
+package inject
+
+import (
+	"github.com/ONSdigital/ras-rm-sample/file-uploader/config"
+	"github.com/ONSdigital/ras-rm-sample/file-uploader/file"
+	"github.com/google/wire"
+	"github.com/spf13/viper"
+	"cloud.google.com/go/pubsub"
+	"context"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var FileProcessor = Inject()
+
+func Inject() file.FileProcessor{
+	wire.Build(NewFileProcessor, ConfigSetup, GenContext, NewPubSub)
+	return file.FileProcessor{}
+}
+
+func ConfigSetup() config.Config {
+	viper.BindEnv("PORT", "8080")
```

Comment:
Should this be SetDefault("PORT", "8080")?

I thought BindEnv just bound environment variable names.

Reply:
Ahh, my mistake, will change